Long test failure

### What happened?

I get a failure when I run long tests, and I noticed that none of the tests affected by the PRs I've submitted run on GitHub.

If that's not done on purpose, it looks like the unit-tests and integration-tests actions do the same thing, perhaps the integration tests could use the `-E long` flag.

### What are the steps to reproduce the bug?

```
git clone git@github.com:ecmwf/earthkit-data.git
mamba env create -n earthkit-data -f tests/environment-unit-tests.yml
conda activate earthkit-data
pip install -e .
python -m pytest -vv -m 'not notebook and not no_cache_init' --cov=. --cov-report=$(COV_REPORT) -E long
```

### Version

0.4.1.dev27+g7a11635

### Platform (OS and architecture)

Darwin MacBook-Pro-3.local 22.6.0 Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:56 PDT 2023; root:xnu-8796.141.3~6/RELEASE_X86_64 x86_64

### Relevant log output

```shell
FAILED tests/sources/test_cds.py::test_cds_non_observation_csv_file_to_pandas_xarray - Exception: the request you have submitted is not valid. There is no data matching your request. Check that you have specified the correct fields and values..
```


### Accompanying data

_No response_

### Organisation

B-Open / CADS-EQC

--- a/tests/sources/test_cds.py
+++ b/tests/sources/test_cds.py
@@ -265,18 +265,20 @@ def test_cds_observation_csv_file_to_pandas_xarray():
     assert list(df.columns)[:2] == ["station_name", "report_timestamp"]
 
 
-@pytest.mark.skip("remote data does not exist")
 @pytest.mark.long_test
 @pytest.mark.download
 @pytest.mark.skipif(NO_CDS, reason="No access to CDS")
 def test_cds_non_observation_csv_file_to_pandas_xarray():
-    collection_id = "sis-energy-derived-reanalysis"
+    collection_id = "sis-energy-derived-projections"
     request = {
+        "format": "zip",
         "variable": "wind_power_generation_onshore",
         "spatial_aggregation": "country_level",
-        "energy_product_type": "energy",
+        "energy_product_type": "capacity_factor_ratio",
         "temporal_aggregation": "daily",
-        "format": "zip",
+        "experiment": "rcp_2_6",
+        "rcm": "hirham5",
+        "gcm": "ec_earth",
     }
     data_cds = from_source("cds", collection_id, **request)
     assert "Date" in data_cds.to_pandas().columns

